Libplanet/Blockchain/BlockChain.cs

```diff
@@ -775,9 +775,25 @@ public long GetNextTxNonce(Address address)
             ).BytesLength;
             int maxBlockBytes = Math.Max(Policy.GetMaxBlockBytes(index), 1);
             var skippedSigners = new HashSet<Address>();
+            var storedNonces = new Dictionary<Address, long>();
+            var nextNonces = new Dictionary<Address, long>();
 
             foreach (Transaction<T> tx in stagedTransactions)
             {
+                if (!storedNonces.ContainsKey(tx.Signer))
+                {
+                    storedNonces[tx.Signer] = Store.GetTxNonce(Id, tx.Signer);
+                }
+
+                if (nextNonces.TryGetValue(tx.Signer, out long prevNonce))
+                {
+                    nextNonces[tx.Signer] = prevNonce + 1;
+                }
+                else
+                {
+                    nextNonces[tx.Signer] = storedNonces[tx.Signer];
```

Comment:
Shouldn't we add 1 to this?

Reply:
It's fine at this moment because I also [changed the condition](https://github.com/planetarium/libplanet/pull/1124/files#diff-ecd6275ea8686534dc160d341079910ede8da3d2ec020236582bdb0906cddc21R831).

but on second thought, adding 1 here is more natural to me too. I'll change it and condition.